src/diff/index.js

```diff
@@ -385,14 +379,3 @@ function doRender(props, state, context) {
 
 	throw error;
 };
-
-export function handleEffects(effects) {
```

Comment:
I just moved this function into the hooks file since it mostly deals with hook data